lib/synced/timestamp.rb

```diff
@@ -0,0 +1,9 @@
+class Synced::Timestamp < ActiveRecord::Base
+  self.table_name = 'synced_timestamps'
+  belongs_to :parent_scope, polymorphic: true
+  scope :with_scope, ->(parent_scope) { where(parent_scope: parent_scope) }
+
+  def self.last_synced_at
+    maximum(:synced_at)
```

Comment:
Well, we could order by `synced_at`, just wondering which solution is faster. With index with ordering expression it won't be really expensive query, `maximum` would still need to compare all dates.